ui/src/views/infra/AddPrimaryStorage.vue

```diff
@@ -162,7 +162,7 @@
         </a-form-item>
         <div
           v-if="form.protocol === 'nfs' || form.protocol === 'SMB' || form.protocol === 'iscsi' || form.protocol === 'vmfs'|| form.protocol === 'Gluster' || form.protocol === 'Linstor' ||
-            (form.protocol === 'PreSetup' && hypervisorType === 'VMware') || form.protocol === 'datastorecluster'">
+            (form.protocol === 'PreSetup' && hypervisorType === 'VMware') || form.protocol === 'datastorecluster' || form.provider === 'Linstor'">
```

Comment:
@rp- any case `form.protocol` = Linstor, and `form.provide`r not? checking based on the updated OR cond check below.

Reply:
In the form it should be possible now to have different protocol/provider, but in the submit part I'll use(overwrite) provider 'Linstor' then.